src/tasks/minions/farmingActivity.ts

```diff
@@ -299,6 +320,10 @@ export const farmingTask: MinionTask = {
 				skillName: SkillsEnum.Woodcutting,
 				amount: Math.floor(woodcuttingXp)
 			});
+			await user.addXP({
+				skillName: SkillsEnum.Herblore,
+				amount: Math.floor(herbloreXp)
+			});
```

Comment:
We should add a xp gain source, for like "Cleaning herbs while farming". The purpose being so we can show users: you gained X amount of xp from cleaning herbs while farming.